src/cymple/internal/declarations/procedure.json

```diff
@@ -0,0 +1,20 @@
+ {
+  "clause_name": "procedure",
+  "methods": [
+    {
+      "name": "procedure",
+      "docstring_summary": "Concatenate a literal procedure",
+      "args": {
+        "literal_procedure": {
+          "type": "str",
+          "description": "A string that evaluated to a cypher procedure"
```

Comment:
did you mean "a string that is evaluated..."?

Reply:
Sure did!